LST-PROD2-MAGIC1.cfg

```diff
@@ -152,23 +152,23 @@ only_triggered_telescopes=1
 % Sampling rate in MHz:
 fadc_MHz = 1640 % MHz sampling rate
 
-fadc_pulse_shape = pulse_CTA-Fx3.dat % Fast pulse shape with tail
+fadc_pulse_shape = pulse_MAGIC_FWHM1.75ns.dat % FWHM of 1.75 ns - like in MAGIC simulations
 
 % Read-out of a 40 ns window (within simulated 55 ns) following the actual trigger:
 fadc_bins = 100       % Number of time intervals simulated for ADC.
 fadc_sum_bins = 50   % Number of ADC time intervals actually summed up or written as trace.
 fadc_sum_offset = 10  % How many intervals summation starts before telescope trigger.
 
 fadc_pedestal = 300          % Per time slice (positive signals only: unsigned!)
-fadc_amplitude = 40.0        % The peak amplitude in a time slice for high gain.
-fadc_noise = 12.0            % Again per time slice (high gain).
+fadc_amplitude = 24.5        % The peak amplitude in a time slice for high gain.
+fadc_noise = 16.1            % Again per time slice (high gain).
 
-num_gains = 2                % Make it clear that we want to use two gains
-fadc_lg_pedestal = 300       % Per time slice (positive signals only: unsigned!)
-fadc_lg_amplitude = 2.2      % The peak amplitude in a time slice for low gain (HG/LG=18 ??)
-fadc_lg_noise = 2.2          % Again per time slice (low gain).
+num_gains = 1                % Make it clear that we want to use two gains
+% fadc_lg_pedestal = 300       % Per time slice (positive signals only: unsigned!)
+% fadc_lg_amplitude = 2.2      % The peak amplitude in a time slice for low gain (HG/LG=18 ??)
+% fadc_lg_noise = 2.2          % Again per time slice (low gain).
 
-fadc_max_signal = 4095       % 12-bit ADC (applies to both channels unless specified)
+fadc_max_signal = 13800       % 12-bit ADC (applies to both channels unless specified)
 fadc_max_sum = 16777215      % 24-bit sum is unlimited for all practical purposes.
```

Comment:
13800 is not 12 bit anymore

Reply:
thx I forgot to change the comment here, I will fix it now